server/serverMethods.js

```diff
@@ -437,6 +439,15 @@ function ServerMethods(aLogLevel, aModules) {
         if (aArchiveAlways) {
           sessionOptions.archiveMode = 'always';
         }
+
+        getAppUsage().then((usage) => {
+          if (usage.lastUpdate + 60000 < Date.now()) {
+            setAppUsage(Date.now(), 1);
+          } else {
+            setAppUsage(usage.lastUpdate, ++ usage.meetings);
```

Comment:
I prefer `usage.meetings++` (although `++usage.meetings` is also OK). If we were linting, this is the rule: https://eslint.org/docs/rules/space-unary-ops.

Reply:
We need to keep it as `++usage.meetings` since if the operator is placed after the variable, the value returned from the operation is the value of the variable before it was incremented and the code breaks (we need the value of usage.meetings after it was incremented).

Removing the space though